node/baremetal/README.md

```diff
@@ -0,0 +1,418 @@
+OP_NODE_L1_ETH_RPC=http://localhost:8545
+
+OP_NODE_L2_ENGINE_AUTH=/opt/base/engine-auth-jwt
+OP_NODE_L2_ENGINE_RPC=http://localhost:8553
+OP_NODE_LOG_LEVEL=info
+OP_NODE_METRICS_ADDR=0.0.0.0
+OP_NODE_METRICS_ENABLED=true
+OP_NODE_METRICS_PORT=7300
+OP_NODE_P2P_AGENT=base
+OP_NODE_P2P_BOOTNODES=enr:-J24QNz9lbrKbN4iSmmjtnr7SjUMk4zB7f1krHZcTZx-JRKZd0kA2gjufUROD6T3sOWDVDnFJRvqBBo62zuF-hYCohOGAYiOoEyEgmlkgnY0gmlwhAPniryHb3BzdGFja4OFQgCJc2VjcDI1NmsxoQKNVFlCxh_B-716tTs-h1vMzZkSs1FTu_OYTNjgufplG4N0Y3CCJAaDdWRwgiQG,enr:-J24QH-f1wt99sfpHy4c0QJM-NfmsIfmlLAMMcgZCUEgKG_BBYFc6FwYgaMJMQN5dsRBJApIok0jFn-9CS842lGpLmqGAYiOoDRAgmlkgnY0gmlwhLhIgb2Hb3BzdGFja4OFQgCJc2VjcDI1NmsxoQJ9FTIv8B9myn1MWaC_2lJ-sMoeCDkusCsk4BYHjjCq04N0Y3CCJAaDdWRwgiQG,enr:-J24QDXyyxvQYsd0yfsN0cRr1lZ1N11zGTplMNlW4xNEc7LkPXh0NAJ9iSOVdRO95GPYAIc6xmyoCCG6_0JxdL3a0zaGAYiOoAjFgmlkgnY0gmlwhAPckbGHb3BzdGFja4OFQgCJc2VjcDI1NmsxoQJwoS7tzwxqXSyFL7g0JM-KWVbgvjfB8JA__T7yY_cYboN0Y3CCJAaDdWRwgiQG,enr:-J24QHmGyBwUZXIcsGYMaUqGGSl4CFdx9Tozu-vQCn5bHIQbR7On7dZbU61vYvfrJr30t0iahSqhc64J46MnUO2JvQaGAYiOoCKKgmlkgnY0gmlwhAPnCzSHb3BzdGFja4OFQgCJc2VjcDI1NmsxoQINc4fSijfbNIiGhcgvwjsjxVFJHUstK9L1T8OTKUjgloN0Y3CCJAaDdWRwgiQG,enr:-J24QG3ypT4xSu0gjb5PABCmVxZqBjVw9ca7pvsI8jl4KATYAnxBmfkaIuEqy9sKvDHKuNCsy57WwK9wTt2aQgcaDDyGAYiOoGAXgmlkgnY0gmlwhDbGmZaHb3BzdGFja4OFQgCJc2VjcDI1NmsxoQIeAK_--tcLEiu7HvoUlbV52MspE0uCocsx1f_rYvRenIN0Y3CCJAaDdWRwgiQG
+OP_NODE_P2P_LISTEN_IP=0.0.0.0
+OP_NODE_P2P_LISTEN_TCP_PORT=9222
+OP_NODE_P2P_LISTEN_UDP_PORT=9222
+OP_NODE_ROLLUP_CONFIG=mainnet/rollup.json
+OP_NODE_RPC_ADDR=0.0.0.0
+OP_NODE_RPC_PORT=8549
+OP_NODE_SNAPSHOT_LOG=/tmp/op-node-snapshot-log
+OP_NODE_VERIFIER_L1_CONFS=4
+
+# OP_NODE_L1_TRUST_RPC allows for faster syncing, but should be used *only* if your L1 RPC node
+# is fully trusted. It also allows op-node to work with clients such as Erigon that do not
+# support storage proofs:
+# OP_NODE_L1_TRUST_RPC=true
+
+OP_NODE_P2P_ADVERTISE_IP=
+```
+
+9. For the OP_NODE_P2P_ADVERTISE_IP, you will need to add the public IP address
+   of your node. You can find this by running `curl ifconfig.me` on the node.
+   Paste the output to the right of the `=` sign.
+
+10. Clone the [Base Node repo](https://github.com/base-org/node):
+
+```bash
+sudo git clone https://github.com/base-org/node.git
+```
+
+11. Copy the mainnet directory out of this repository. This contains a genesis
+    and rollup config file for the OP-Geth and OP-Node binaries:
+
+```bash
+sudo cp -r node/mainnet .
+```
+
+12. Init the OP-Geth data:
+
+```bash
+sudo ./geth --verbosity=3 init --datadir=/opt/basedata/geth/chaindata mainnet/genesis-l2.json
+```
+
+This should initialize the data folder with the genesis block and correct chain
+data for OP-Geth to run as Base.
+
+13. There needs to be a JWT secret for OP-Geth to run. Place the following in a
+    file called `/opt/base/engine-auth-jwt`:
+
+```bash
+688f5d737bad920bdfb2fc2f488d6b6209eebda1dae949a8de91398d932c517a
```

Comment:
Have them generate a new one

Reply:
the https://github.com/base-org/node has this hardcoded. Felt like it's fine if they're hardcoding this there. It's a local system anyways.